### PR TITLE
Version 2 #101

### PR DESCRIPTION
**Pull request by ericgaspar**
Allow to configure extra nginx conf in $app.d/ directory  #101

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -108,62 +60,39 @@ fi
 if [ -n "$(ynh_app_setting_get --app=$app --key=user)" ]
 then
     ynh_systemd_action --service_name=php${phpversion}-fpm --action=stop
-    ynh_system_user_delete --username="$(ynh_app_setting_get --app=$app --key=user)"
+	ynh_system_user_delete --username="$(ynh_app_setting_get --app=$app --key=user)"
```
Comment (alexAubin) [inline scripts/upgrade:63]:
```suggestion
    ynh_system_user_delete --username="$(ynh_app_setting_get --app=$app --key=user)"
```

**Comment by alexAubin** [comment]
Also yes : we can definitely remove the old actions folder which ain't used anyway ..

Regarding the "if mysql" block in the remove script, to me it sounds like we should keep it

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 2]
```diff
@@ -9,70 +9,33 @@
...
-
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
 ynh_script_progression --message="Storing installation settings..."
 
-ynh_app_setting_set --app=$app --key=domain --value=$domain
-ynh_app_setting_set --app=$app --key=path --value=$path_url
+ynh_app_setting_set --app=$app --key=password --value=$password
+ynh_app_setting_set --app=$app --key=ssh_port --value=$ssh_port
```
[the remark this answers is not in the document]
Comment (alexAubin) [reply scripts/install]: Ah yes @_@ but hmf having just "22 (unless you changed the port)" is good enough imho

Clearly using dynamic-values from the general system and independant of the app is a use case not well covered ..

**Review by alexAubin — APPROVED** [review]
LGTM, cheers, thanks for all the work !

**Pull request merged**